.gitmodules

```diff
@@ -4,7 +4,8 @@
 	branch = v2.4.3
 [submodule "freertos_kernel"]
 	path = freertos_kernel
-	url = https://github.com/FreeRTOS/FreeRTOS-Kernel.git
+	url = https://github.com/shubhamkulkarni97/FreeRTOS-Kernel.git
```

Comment:
Since the kernel port has been merged here https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/231, can you change the kernel submodule back to the default?

Reply:
Changed the kernel submodule back to default